aspnetcore/security/anti-request-forgery.md

```diff
@@ -4,142 +4,230 @@ author: steve-smith
 
-Note: In this context, *session* refers to the client-side session during which the user is authenticated. It's unrelated to server-side sessions or [session middleware](xref:fundamentals/app-state).
+Users can guard against CSRF vulnerabilities by taking simple precautions:
 
-Users can guard against CSRF vulnerabilities by:
-* Logging off of web sites when they have finished using them.
-* Clearing their browser's cookies periodically.
+* Sign off of web apps when finished using them.
+* Clear browser cookies periodically.
 
 However, CSRF vulnerabilities are fundamentally a problem with the web app, not the end user.
 
-## How does ASP.NET Core MVC address CSRF?
+## Authentication fundamentals
 
-> [!WARNING]
-> ASP.NET Core implements anti-request-forgery using the [ASP.NET Core data protection stack](xref:security/data-protection/introduction). ASP.NET Core data protection must be configured to work in a server farm. See [Configure ASP.NET Core Data Protection](xref:security/data-protection/configuration/overview) for more information.
+Cookie-based authentication is a popular form of authentication. Token-based authentication systems are growing in popularity, especially for Single Page Applications (SPAs).
+
+### Cookie-based authentication
+
+When a user authenticates using their username and password, they're issued a token, containing an authentication ticket that can be used for authentication and authorization. The token is stored as a cookie that accompanies every request the client makes. Generating and validating this cookie is performed by the Cookie Authentication Middleware. The [middleware](xref:fundamentals/middleware/index) serializes a user principal into an encrypted cookie. On subsequent requests, the middleware validates the cookie, recreates the principal, and assigns the principal to the [User](/dotnet/api/microsoft.aspnetcore.http.httpcontext.user) property of [HttpContext](/dotnet/api/microsoft.aspnetcore.http.httpcontext).
 
-ASP.NET Core anti-request-forgery default data protection configuration 
+When a user is authenticated to access an app, a server-side user session is created and stored in a database or some other persistent store. The app generates a session key that points to the stored session in the data store and includes it in the authentication ticket/cookie. The web server checks this session key each time a user requests a resource that requires authorization. The app checks whether the associated user session has sufficient privileges to access the requested resource. If the user is authorized to access the resource, the request continues. Otherwise, the request returns as unauthorized. Cookies are used to make the app appear stateful, since the app is able to maintain the user's authentication/authorization state on each request.
```

Comment:
Session is optional, if we mean session in its classical sense, which seems to be the case here. Cookie auth persists the entire identity in the cookie, it doesn't need sessions. Wording here is confusing.

Reply:
I feel like we could delete the paragraph in this context because the topic pertains to "antirequest forgery," while this paragraph is just providing general background info.